### PR TITLE
Add utility to generate BZL containing tool shas

### PR DESCRIPTION
**Pull request by acmcarther**
This adds a util that can generate a BZL file that contains all known shas by downloading the `sha256` file from static.rust-lang.org.

This isn't particularly useful _yet_, but it will be necessary if we support fetching stdlibs separately from the compiler (e.g. rust-0.0.0-$TARGET versus rustc-(...) and rust-std-(...)). This also opens the door to bringing in other tools such as rustfmt and rls.

ctx: https://github.com/bazelbuild/rules_rust/issues/101#issuecomment-404624169

@mfarrugi: Are you up to reviewing this?

### CODE REVIEW

File: util/fetch_shas.sh [hunk 1]
```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+# Enumerates the list of expected downloadable files, loads the SHAs for each file, then
+# dumps the result to stdout
+#
+# SHould be run from workspace root.
```
Comment (mfarrugi) [inline util/fetch_shas.sh]: Should :)

File: util/fetch_shas_VERSIONS.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+1.26.0
+1.26.1
+1.26.2
+1.27.0
```
Comment (mfarrugi) [inline util/fetch_shas_VERSIONS.txt]: 1.27.1 just released

Comment (acmcarther) [reply util/fetch_shas_VERSIONS.txt]: Added.

File: util/fetch_shas.sh (hunk 1 above)
Comment (acmcarther) [reply util/fetch_shas.sh]: Updated

**Pull request merged**